guides/common/modules/proc_registering-a-host.adoc

```diff
@@ -94,87 +87,16 @@ The following is an example of the `curl` command with the `--insecure` option:
 ----
 # curl -sS --insecure https://{foreman-example-com}/register ...
 ----
-. Select the *Advanced* tab.
-. Optional: From the *Setup REX* list, select whether you want to deploy {Project} SSH keys to hosts or not.
-+
-If set to `Yes`, public SSH keys will be installed on the registered host.
-The inherited value is based on the `host_registration_remote_execution` parameter.
-It can be inherited, for example from a host group, an operating system, or an organization.
-When overridden, the selected value will be stored on host parameter level.
-ifdef::client-content-dnf[]
-. Optional: From the *Setup Insights* list, select whether you want to install `insights-client` and register the hosts to Insights.
-+
-The Insights tool is available for {RHEL} only.
-It has no effect on other operating systems.
-+
-You must enable the following repositories on a registered machine:
-
-* {RHEL} 6: `rhel-6-server-rpms`
-* {RHEL} 7: `rhel-7-server-rpms`
-* {RHEL} 8: `rhel-8-for-x86_64-appstream-rpms`
-+
-The `insights-client` package is installed by default on {RHEL} 8 except in environments whereby {RHEL} 8 was deployed with "Minimal Install" option.
-endif::[]
-. Optional: In the *Install packages* field, list the packages (separated with spaces) that you want to install on the host upon registration.
-This can be set by the `host_packages` parameter.
-. Optional: Select the *Update packages* option to update all packages on the host upon registration.
-This can be set by the `host_update_packages` parameter.
-. In the *Repositories* field, click *Add repositories for registration*.
-+
-On the *Repository list* window, add content that is required before performing the registration.
-For example, it can be useful to make the `subscription-manager` package available for the purpose of the registration.
-+
-* In the *Repository* field, enter a repository to be added before the registration is performed.
-ifdef::client-content-dnf[]
-For Red Hat family distributions, enter the URL of the repository, for example `\http://rpm.example.com/`.
-endif::[]
-ifdef::client-content-apt[]
-For Debian OS families, enter the whole line of list file content, for example `deb \http://deb.example.com/ bookworm 1.0`.
-endif::[]
-* Optional: In the *Repository GPG key URL* field, specify the public key to verify the signatures of GPG-signed packages.
-It needs to be specified in the ASCII form with the GPG public key header.
-ifdef::katello,orcharhino,satellite[]
```

Comment:
I'd like to keep this; at least for non-satellite builds. IMHO it add value because it explains how to insert the URLs and what type of GPG pub key URL.

Also, line 90" in the repository field" is wrong because by now, it's a set of repositories.

Foreman 3.12:

![image](https://github.com/user-attachments/assets/65ad7bb1-32f0-4f9d-80c0-49c31346e3ba)

If you argue that we can drop it because the tooltip explains the fields sufficiently, then I'm also OK.

Reply:
> If you argue that we can drop it because the tooltip explains the fields sufficiently, then I'm also OK.

Yes :) For transparency (and to show that I did my due diligence and am not just deleting things on a whim :)), this is the tooltip for the `Repository` field:

`A repository to be added before the registration is performed. For Red Hat and SUSE family distributions, this should be the URL of the repository, e.g. 'http://rpm.example.com/'. For Debian OS families, it's the whole list file content, e.g. 'deb http://deb.example.com/ buster 1.0'.`

Tooltip for the `Repository GPG key URL` field:

`If packages are GPG signed, the public key can be specified here to verify the packages signatures. It needs to be specified in the ascii form with the GPG public key header.
`

Information-wise, these tooltips match what we currently document. Wording-wise, they don't follow our style guides but that doesn't justify duplicating the information.

Reply:
> Also, line 90" in the repository field" is wrong because by now, it's a set of repositories.

Good catch! I changed the line to refer to `Repositories` field.